doc/FAQ

```diff
@@ -0,0 +1,101 @@
+- NodeOS - Lightweight operating system using Node.js as userspace - FAQ -
+--------------------------------------------------------------------------
+
+
+1. NodeOS is a fantastic project, what can I do to help it?
+
+- Linux kernel programmers
+- Node.js programmers
+- Designers
+- Testers
+
+
+2. How to run NodeOS on my computer?
+
+- Running it on Qemu
+-- Download a prebuild version at https://github.com/NodeOS/NodeOS/releases
+-- Uncompress and execute run.sh
+-- Ports maped on your system to Qemu (50080 => 80, 50443 => 443)
+
+- Running it on Docker
+-- http://node-os.com/blog/get-involved
+-- Get NodeOS to access the internet
+--- ...
+
+
+3. NodeOS is a complex project, it is made by a lot of other projects. What are they? 
+
+ You may find your self helping NodeOS by pull-requesting in one of the following projects on https://github.com/
+
+- NodeOS/NodeOS
+- groundwater/node-bin-nsh
+- piranna/Davius
+- piranna/nodeos-reverse-proxy
+- piranna/nodeos-reverse-proxy-register
+- piranna/logon
+- piranna/coreutils.js
+- piranna/Gaia
+- ...
+
+
+4. I'm going to start my contributions. What about code styles?
+
+ It is not set on stone, but your pull-requests have more chance to get clearance if you follow some simple rules.
+
+- Ident with tabs
+- If steatment 
+    if (true) {
+        alert('Good');
+    } else {
+        alert("Bad");
+    }
+- Case steatment
+    switch (variable) {
+        case 'One':
+            alert('Good');
+            break;
+
+        default:
+            alert('Ok');
+    }
```

Comment:
Totally wrong. There's the [NodeOS standard](https://www.npmjs.com/package/nodeos-standard) module but needs to be integrated better.